### PR TITLE
refactor(doctor): Clean up Dead Stuff

### PR DESCRIPTION
**Pull request by johnludwigm**
Was looking at the code and figured I'd run the standard clean-up stuff.

* Deleted commented-out code

### CODE REVIEW

File: doctor/lib/mojibake.py [hunk 1]
```diff
@@ -47,7 +47,6 @@ def fix_mojibake(text):
         "Ò": "N",
         "Ñ": "O",
         "Ð": "P",
-        "": "Q",  # Missing
```
Comment (johnludwigm) [inline doctor/lib/mojibake.py:50]: Key is overwritten in L56

Comment (mlissner) [reply doctor/lib/mojibake.py:50]: That's actually deliberate to document how we don't have the replacement keys for Q or X.

**Review by mlissner — CHANGES_REQUESTED** [review]
Looks good, but I'd keep the two empty keys as code-as-documentation.

File: doctor/lib/mojibake.py (hunk 1 above)
Comment (johnludwigm) [reply doctor/lib/mojibake.py]: @mlissner fixed now. Should be good to go.

**Review by mlissner — APPROVED** [review]

**Comment by mlissner** [comment]
Thanks

**Pull request merged**